### ISSUE TITLE
Thunderbird Appointment should propose a solution to set all calendars set by the user in Thunderbird app

### ISSUE THREAD

**Issue by NicolasWeb** [thunderbird/appointment#715]
**Is your feature request related to a problem? Please describe.**
I have configured calendars in Thunderbird Desktop and on my mobile phone (CalDAV) and can't have them at once while using Thunderbird Appointment

**Describe the solution you'd like**
As a user, I want Thunderbird Appointment to propose me by default all calendars I set in Thunderbird (Desktop/Mobile_in_the_future/Mozilla Account)

**Describe alternatives you've considered**
Manually set again my Thunderbird's calendar setup. Set one by one all my calendars configured in Thunderbird app.

**Comment by Sancus** [comment]
Interesting idea, Appointment doesn't really have a way to know what you've configured in the native clients.

This could be done as an integration with the Pro add-on, though, and it could simplify sign-up.

**Comment by malini** [comment]
yes, this is a good integration point for the Pro add-on, thanks for the submission!

A few things have to be in place before, namely a) making a functional add-on for this and b) having a single TB Pro add-on instead of individual ones (ideally). As a result, I'm setting this as a feature for post-MVP as we don't have TB add-on as a requirement for the MVP of Appointment.
